cardano-node.cabal

```diff
@@ -11,43 +11,70 @@ cabal-version:  >= 1.10
 extra-source-files: README.md, ChangeLog.md
 
 library
+  hs-source-dirs:      src
+
   exposed-modules:
-      Lib
+                       Lib
+
   other-modules:
-      Paths_cardano_node
-  hs-source-dirs:
-      src
+                       Paths_cardano_node
+
   build-depends:
-      base >=4.7 && <5
-    , cardano-ledger
-    , cardano-prelude
-    , cardano-shell
-    , ouroboros-consensus
-    , ouroboros-network
-  default-language: Haskell2010
+                       base >=4.7 && <5
```

Comment:
I am almost certain this will not build with `base == 4.7.*`. Since ghc 8.6 uses `base` version `4.12`, maybe that should be `base >=4.12 && <5`.